tests/unit/test_file_handle_cache.py

```diff
@@ -0,0 +1,52 @@
+"""
+Tests the file handle cache
+"""
+
+from ga4gh import datamodel
```

Comment:
Please change this import to `import ga4gh.datamodel as datamodel`, and move to below the standard library imports. (See the STYLE.md file for details on this completely arbitrary convention)